cgi-bin/LJ/S2/EntryPage.pm

```diff
@@ -555,7 +555,7 @@ sub EntryPage_entry
     $comments->{show_readlink} &&= $get_mode eq 'reply';
 
     my $subject = LJ::CleanHTML::quote_html( $entry->subject_html, $get->{nohtml} );
-    my $event = LJ::CleanHTML::quote_html( $entry->event_html, $get->{nohtml} );
+    my $event = LJ::CleanHTML::quote_html( $entry->event_html( { proxy_insecure_content => $LJ::IS_SSL } ), $get->{nohtml} );
```

Comment:
By the logic of this patch, don't we just want to have `proxy_insecure_content` be based on `$LJ::IS_SSL` in the functions? That way you don't have to pass it in?